components/org.wso2.carbon.identity.application.authenticator.fido2.endpoint/src/main/java/org/wso2/carbon/identity/application/authenticator/fido2/endpoint/common/Util.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
```

Comment:
Unnecessary `WSO2 Inc. ` segment in the header

Reply:
Isn't this the recommended license header template?

`Copyright (c) $today.year, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.

WSO2 Inc. licenses this file to you under the Apache License,
Version 2.0 (the "License"); you may not use this file except
in compliance with the License.
You may obtain a copy of the License at

http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing,
software distributed under the License is distributed on an
"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
KIND, either express or implied.  See the License for the
specific language governing permissions and limitations
under the License.`

Reply:
Please disregard my 1st comment. However, this is different than what's been used in some other files. eg: `CredentialIdApiServiceImpl.java`, `FIDO2Constants.java`. Better to make them consistent. The one used in this class is the correct one.

Reply:
fixed: https://github.com/wso2-extensions/identity-local-auth-fido/pull/50